spec/support/user_helper.rb

```diff
@@ -0,0 +1,21 @@
+def login(email)
+  visit root_path
+  fill_in :email, with: email
+  fill_in :password, with: 12345
+  click_button 'Log In'
+end
+
+def create_card
+  click_link 'Добавить карточку'
+  fill_in :card_original_text, with: 'Hello'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.